dags/common/sftp_service.py

```diff
@@ -22,6 +23,7 @@ def __init__(
         self.port = port
         self.logger = get_logger().bind(class_name=type(self).__name__)
         self.dir = dir
+        self.file_names = []
```

Comment:
we don't need that

Reply:
but we have to save file paths somewhere, that the call back store_files_name could reach it

Reply:
changed